lib/commands/login/index.js

```diff
@@ -0,0 +1,61 @@
+import isInteractive from 'is-interactive'
+import meow from 'meow'
+import ora from 'ora'
+import prompts from 'prompts'
+
+import { AuthError, InputError } from '../../utils/errors.js'
+import { setupSdk } from '../../utils/sdk.js'
+import { getSetting, updateSetting } from '../../utils/settings.js'
+
+const description = 'Socket API login'
+
+/** @type {import('../../utils/meow-with-subcommands').CliSubcommand} */
+export const login = {
+  description,
+  run: async (argv, importMeta, { parentName }) => {
+    const name = parentName + ' login'
+    const cli = meow(`
+      Usage
+        $ ${name}
+
+      Logs into the Socket API by prompting for an API key
+
+      Examples
+        $ ${name}
+    `, {
+      argv,
+      description,
+      importMeta,
+    })
+
+    if (cli.input.length) cli.showHelp()
+
+    if (!isInteractive()) {
+      throw new InputError('cannot prompt for credentials in a non-interactive shell')
+    }
+    const { apiKey } = await prompts({
+      type: 'password',
+      name: 'apiKey',
+      message: 'Enter your Socket.dev API key',
```

Comment:
this should include a rough link to the basic dashboard somehow or the docs so a user could find where the API key comes from